packages/nodes-base/nodes/EmailReadImap/EmailReadImap.node.ts

```diff
@@ -437,6 +437,12 @@ export class EmailReadImap implements INodeType {
 				};
 			}
 
+			if (credentials.secure) {
+				config.imap.tlsOptions = {
+					servername: credentials.host as string,
```

Comment:
A few lines above we have another block that sets the `config.imap.tlsOptions` attribute.

Maybe we should be merging these since they override each other. WDYT?